### PR TITLE
[Feature][*] Change dev branch version to 1.0.0-SNAPSHOT

### LINKED ISSUE #2091: [Feature][*] Change dev branch version to 1.0.0-SNAPSHOT
### Search before asking

- [X] I had searched in the [issues](https://github.com/DataLinkDC/dinky/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

- [ ] all pom file version change to 1.0.0-SNAPSHOT
- [ ] script/sql/upgrade/0.8   -> change to 1.0.0-SNAPSHOT
- [ ] script/sql/dinky.sql -> change dinky-mysql.sql
- [ ] `dinky-web/src/services/constants.tsx` of const VERSION value change to 1.0.0-SNAPSHOT

### Use case

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by Jrebel-i**
## Purpose of the pull request

close https://github.com/DataLinkDC/dinky/issues/2091

## Brief change log

 - [x] all pom file version change to 1.0.0-SNAPSHOT
 - [x] script/sql/upgrade/0.8 -> change to 1.0.0-SNAPSHOT
 - [x] script/sql/dinky.sql -> change dinky-mysql.sql
 - [x] dinky-web/src/services/constants.tsx of const VERSION value change to 1.0.0-SNAPSHOT

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

File: dinky-assembly/src/main/assembly/package.xml [hunk 1]
```diff
@@ -54,7 +54,7 @@
             <directory>${project.parent.basedir}/script/sql</directory>
             <outputDirectory>sql</outputDirectory>
             <includes>
-                <include>dinky.sql</include>
+                <include>dinky-mysql.sql</include>
```
Comment (Zzm0809) [inline dinky-assembly/src/main/assembly/package.xml:57]: please append  `<include>dinky-pg.sql</include>`

**Comment by Zzm0809** [comment]
hi, @Jrebel-i  Regarding the documentation module, you only need to modify the information about the sql file in the next (docs/) version, and the others belong to the historical version and do not follow the changes in the dev branch documentation

**Comment by Jrebel-i** [comment]
> hi, @Jrebel-i Regarding the documentation module, you only need to modify the information about the sql file in the next (docs/) version, and the others belong to the historical version and do not follow the changes in the dev branch documentation
hi, @zhu-mingye May I ask if this modification is possible?

File: docs/docs/developer_guide/local_debug.md [hunk 2]
```diff
@@ -165,7 +165,7 @@ spring:
 
 ### 初始化数据库
 
-在MySQL数据库创建 dlink 用户并在 dlink 数据库中执行 dlink-doc/sql/dinky.sql 文件。此外 dlink-doc/sql/upgrade 目录下存放了了各版本的升级 sql 请依次按照版本号执行。
+在MySQL数据库创建 dlink 用户并在 dlink 数据库中执行 dlink-doc/sql/dinky-mysql.sql 文件。此外 dlink-doc/sql/upgrade 目录下存放了了各版本的升级 sql 请依次按照版本号执行。
```
Comment (Zzm0809) [inline docs/docs/developer_guide/local_debug.md]: change `dlink-doc/sql/dinky-mysql.sql` to `script/sql/dinky-mysql.sql`

File: docs/docs/developer_guide/remote_debug.md [hunk 3]
```diff
@@ -157,7 +157,7 @@ sa-token:
 
 ### 初始化数据库
 
-在MySQL数据库创建 dlink 用户并在 dlink 数据库中执行 dlink-doc/sql/dinky.sql 文件。此外 dlink-doc/sql/upgrade 目录下存放了了各版本的升级 sql 请依次按照版本号执行。
+在MySQL数据库创建 dlink 用户并在 dlink 数据库中执行 dlink-doc/sql/dinky-mysql.sql 文件。此外 dlink-doc/sql/upgrade 目录下存放了了各版本的升级 sql 请依次按照版本号执行。
```
Comment (Zzm0809) [inline docs/docs/developer_guide/remote_debug.md]: change `dlink-doc/sql/dinky-mysql.sql` to `script/sql/dinky-mysql.sql`

**Review by zackyoungh — COMMENTED** [review]
还差web界面显示的0.8.00

**Comment by Zzm0809** [comment]
> 还差web界面显示的0.8.00

这个改过了

**Review by Zzm0809 — APPROVED** [review]
LGTM

**Pull request merged**